pkg/kubelet/apis/config/v1beta1/defaults.go

```diff
@@ -252,4 +252,10 @@ func SetDefaults_KubeletConfiguration(obj *kubeletconfigv1beta1.KubeletConfigura
 	if obj.EnableDebugFlagsHandler == nil {
 		obj.EnableDebugFlagsHandler = utilpointer.BoolPtr(true)
 	}
+	if obj.ShutdownGracePeriod == zeroDuration {
+		obj.ShutdownGracePeriod = metav1.Duration{Duration: 0}
```

Comment:
How is `metav1.Duration{Duration: 0}` different from `zeroDuration` ?  They seem identical to me?

Reply:
It should be the same, I just added it to be explicit that we actually want to the default value here (since all the existing kubelet config time duration fields compared against zeroDuration here and set the appropriate default). 

So I think it can be dropped, but was thinking would be better to be explicit about the default.